cmd/tailscale/cli/netcheck.go

```diff
@@ -52,9 +52,15 @@ func runNetcheck(ctx context.Context, args []string) error {
 	if err != nil {
 		return err
 	}
+
+	// Ensure that we close the portmapper after running a netcheck; this
+	// will release any port mappings created.
+	pm := portmapper.NewClient(logf, netMon, nil, nil, nil)
```

Comment:
that signature 😂 

Maybe we can also [later?] add a `pm.SetDiscoverOnly(true)` method to it that asks that we prefer to only discover port mappability without making mappings?

Reply:
Yeah, that's a good idea for a follow-up 👍